addons/sys_intercom/fnc_getStationVariableName.sqf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Author: ACRE2Team
+ * Gets the variable name for accessing intercom configuration in the players position.
+ *
+ * Arguments:
+ * 0: Vehicle with intercom <OBJECT>
+ * 1: Unit to be checked <OBJECT>
+ *
+ * Return Value:
+ * Variable name, string of length 0 if not found <STRING>
+ *
+ * Example:
+ * [vehicle acre_player, acre_player] call acre_sys_intercom_fnc_getStationVariableName
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle", "_unit"];
+
+private _found = false;
+private _varName = QGVAR(station_);
+{
+    if (_unit isEqualTo (_x select 0)) then {
```

Comment:
Again `exitWith`.

Reply:
Here found variable must be used for checking afterwards.

Reply:
Where? It's only used to set to `""` if `!_found`. But you don't need that if you invert the logic like I suggested with other comments on this file.

Reply:
you are right. long day.

Reply:
You are right. Long day.